lib/vdsm/tool/configurators/libvirt.py

```diff
@@ -99,6 +100,9 @@ def isconfigured():
     """
     Check if libvirt is already configured for vdsm
     """
+    # Creates libvirt conf file if not exists
```

Comment:
Strictly speaking, this is not really "libvirt configration" - this one can be found in `/etc/libvirt/libvirt.conf`. Let's call it "system-level libvirt config". This applies to the commit message too.